docs/root/api/http.rst

```diff
@@ -17,15 +17,15 @@ a previously created :ref:`Envoy instance <api_starting_envoy>`.
 
 **Kotlin**::
 
-  val request = RequestBuilder(RequestMethod.POST, "https", "api.envoyproxy.io", "/foo")
+  val request = RequestBuilder(RequestMethod.POST, "https", "api.envoyproxy.io", "/foo", upstreamHttpProtocol = UpstreamRequestProtocol.HTTP2)
     .addRetryPolicy(RetryPolicy(...))
     .addHeader("x-custom-header", "foobar")
     ...
     .build()
 
 **Swift**::
 
-  let request = RequestBuilder(method: .post, scheme: "https", authority: "api.envoyproxy.io", path: "/foo")
+  let request = RequestBuilder(method: .post, scheme: "https", authority: "api.envoyproxy.io", path: "/foo", upstreamHttpProtocol: .http2)
```

Comment:
I'd suggest moving this out of the constructor, and adding it via a builder method.

Reply:
+1 to this suggestion. `addUpstreamProtocol`?